package.json

```diff
@@ -13,17 +13,19 @@
     "dist"
   ],
   "scripts": {
-    "test": "mocha --compilers js:./babel-register --inline-diffs --check-leaks --ui tdd --reporter dot test",
-    "build": "babel --modules commonStrict --source-maps-inline --out-dir dist src",
+    "test": "mocha --compilers js:babel-core/register --inline-diffs --check-leaks --ui tdd --reporter dot test",
+    "build": "babel --source-maps-inline --out-dir dist src",
     "prepublish": "rm -rf dist && npm update && npm run build"
   },
   "dependencies": {
-    "shift-spec": "^2015.1.3"
+    "shift-spec": "2015.2.0"
```

Comment:
Ah, no, this is just because "^2015.2.0" [matches](https://docs.npmjs.com/misc/semver#caret-ranges-1-2-3-0-2-5-0-0-4) "2015.1.3". This should instead be "2015.2.x".